examples/solid/src/routes/api/uploadthing/core.ts

```diff
@@ -1,7 +1,8 @@
 import { createUploadthing, type FileRouter } from "uploadthing/server";
+
 const f = createUploadthing();
 
-const auth = (req: Request) => ({ id: "fakeId" }); // Fake auth function
+const auth = (req: Request) => Promise.resolve({ id: "fakeId" }); // Fake auth function
```

Comment:
should this function be marked async?

Reply:
Linter yells since there are no awaits

Reply:
? 
![image](https://github.com/pingdotgg/uploadthing/assets/11494384/81b2a06b-2dca-454d-948c-6852ddd8afd3)

Reply:
What I mean is doing 

const auth = async => ({ ... });

Will result in the linter yelling at you cause your async function has no await. Doing the Promise.resolve thing makes it so the function is returning a promise (and thus can be awaited in the middleware like we want) without this lint warning.

Really a nit thing, can also just disable the linter if that's more to your liking

Reply:
Oh I see now. My bad. The way you have it is fine 👍